### ISSUE TITLE
[Bug] Focusing on Custom Actors

### ISSUE THREAD

**Issue by Myterian** [FlaxEngine/FlaxEngine#2564]
**Issue description:**
Selecting a `custom actor` and focusing on them via pressing `F` will zoom to the Scene Origin, instead of the custom actor.

**Steps to reproduce:**
1. Create a custom Actor
2. Create instance in scene hierarchy
3. Select and focus

**Minimal reproduction project:**
Basic

**Flax version:**
1.8.6511.1

**Comment by Myterian** [comment]
No, it's using `c#` and currently not setting `_sphere `or `_box`

**Comment by lifeformed** [comment]
My workaround in C# is to derive from EmptyActor instead of Actor, since EmptyActor has Sphere and Box updated OnTransformChanged.

**Comment by GasimoCodes** [comment]
I believe the default behaviour should be to set the focus position to be the current Transform position when bounds are not set. Creating custom actors or even using some built-in actors can still move Camera to (0,0,0) instead of the object transform.

**Comment by Myterian** [comment]
Transform position can be different from the objects center. It makes sense to zoom to the center when focusing on a mesh, for example, cause you want to edit it as a whole, not just at the bottom
